components/ActionSheet/examples/ActionSheetComparisonExampleViewController.m

```diff
@@ -27,61 +30,48 @@ @interface ActionSheetComparisonExampleViewController ()
 
 @end
 
-@implementation ActionSheetComparisonExampleViewController {
-  MDCButtonScheme *_buttonScheme;
-}
+@implementation ActionSheetComparisonExampleViewController
 
 - (instancetype)init {
   self = [super init];
   if (self) {
     self.title = @"Action Sheet";
-    _colorScheme = [[MDCSemanticColorScheme alloc] init];
-    _typographyScheme = [[MDCTypographyScheme alloc] init];
+    _containerScheme = [[MDCContainerScheme alloc] init];
+    _containerScheme.colorScheme =
+        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
     _showMaterialButton = [[MDCButton alloc] init];
     _showUIKitButton = [[MDCButton alloc] init];
-    _buttonScheme = [[MDCButtonScheme alloc] init];
   }
   return self;
 }
 
-- (MDCContainerScheme *)containerScheme {
-  if (!_containerScheme) {
-    _containerScheme = [[MDCContainerScheme alloc] init];
-  }
-  _containerScheme.colorScheme = self.colorScheme;
-  _containerScheme.typographyScheme = self.typographyScheme;
-  return _containerScheme;
-}
-
 - (void)viewDidLoad {
   [super viewDidLoad];
 
-  self.view.backgroundColor = _colorScheme.backgroundColor;
-  [_showMaterialButton setTitle:@"Show Material Action sheet" forState:UIControlStateNormal];
-  [_showMaterialButton sizeToFit];
-  [_showUIKitButton setTitle:@"Show UIKit Action sheet" forState:UIControlStateNormal];
-  [_showUIKitButton sizeToFit];
-  _buttonScheme.colorScheme = _colorScheme;
-  _buttonScheme.typographyScheme = _typographyScheme;
-  [MDCContainedButtonThemer applyScheme:_buttonScheme toButton:_showMaterialButton];
-  [_showMaterialButton addTarget:self
+  self.view.backgroundColor = self.containerScheme.colorScheme.backgroundColor;
```

Comment:
Please add a conditional check and provide a default for `colorScheme`.

Reply:
Fixed